custom-routes-vis/src/main/webapp/script.js

```diff
@@ -339,7 +339,7 @@ function selectRouteDisplayDetails(routeNum, totalDurationSec, totalDistanceMete
 
   if (routeToken !== '') {
     routeInfoElement.innerText += '\nRoute Token: check the developer console';
```

Comment:
I thought we discussed you adding a "copy to clipboard" button here for the route token?